index.ts

```diff
@@ -0,0 +1,8 @@
+import validateCaPostalCodes from './src/ca-postal-codes';
+import validateUsPostalCodes from './src/ca-postal-codes';
```

Comment:
`index.ts` should be in the `src` directory.

Also

`import validateUsPostalCodes from './src/ca-postal-codes';` should be `import validateUsPostalCodes from './src/us-zip-codes';`